hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/storage/AsyncInventoryStorage.java

```diff
@@ -546,17 +546,28 @@ private void removeResources(List<QueueElement> removeQueueElements) throws Exce
             StringBuilder url = Util.getContextUrlString(AsyncInventoryStorage.this.config.getUrl(),
                     AsyncInventoryStorage.this.config.getInventoryContext());
             for (QueueElement resourceElement : removeQueueElements) {
+
                 StringBuilder deleteUrl = new StringBuilder(url.toString());
-                deleteUrl.append("feeds/").append(resourceElement.getFeedId()).append("/resources");
-                for (Resource<?> resource : getAncestry(resourceElement.getResource())) {
-                    deleteUrl.append('/');
-                    deleteUrl.append(Util.urlEncode(resource.getID().getIDString()));
+
+                List< Resource<?> > ancestries = getAncestry(resourceElement.getResource());
+                String tenandId = resourceElement.getTenantId();
+                StringBuilder resourcePath = new StringBuilder();
+                Collections.reverse(ancestries);
+                for (Resource<?> resource : ancestries) {
+                    String resourceIdPath = "/" + Util.urlEncode(resource.getID().getIDString());
+                    resourcePath.insert(0, resourceIdPath);
                 }
+                CanonicalPath pathPrefix = CanonicalPath.of().tenant(tenandId).feed(feedId).get();
+                CanonicalPath resourceCanonicalPath = CanonicalPath.fromPartiallyUntypedString(resourcePath.toString(),
+                        pathPrefix, org.hawkular.inventory.api.model.Resource.class);
+
+                deleteUrl.append("entity").append(resourceCanonicalPath.toString());
 
```

Comment:
can you add a small one-line comment here just to give an example of what the final url is going to look like? Just to give someone reading the code an idea of what the inventory REST URL is expected to look like